cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/reducers/Umc.reducer.tsx

```diff
@@ -0,0 +1,9 @@
+// App imports
```

Comment:
Rather than introduce a new reducer, please re-use the Wizard.reducer.tsx (which I added in my last PR to avoid a proliferation of identical reducers)